addons/bottom-navigation-bar/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Bottom navigation bar",
+  "description": "Positions the navigation bar at the bottom of the page.",
+  "tags": ["community", "theme"],
+  "versionAdded": "1.27.2",
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["https://scratch.mit.edu/*"]
+    },
+    {
+      "url": "infinite-scroll.css",
+      "matches": ["https://scratch.mit.edu/*"],
+      "if": {
+        "addonEnabled": ["infinite-scroll"]

```

Comment:
It doesn't work when you have `infinite-scroll` addon enabled, but you unselected fix footer setting.

Reply:
Also, you have this ugly blue bar at bottom of page
![Screenshot from 2022-07-23 20-21-09](https://user-images.githubusercontent.com/89837724/180617888-6133b49f-ff3c-44de-8b56-ad78b2b39af2.png)